bundling/bundle-web.ts

```diff
@@ -0,0 +1,41 @@
+import { bundle } from "https://deno.land/x/emit@0.12.0/mod.ts";
+import { createCache } from "https://deno.land/x/deno_cache@0.4.1/mod.ts";
+
+// Parse args
+const [release, source = `https://deno.land/x/grammy@${release}/mod.ts`] =
+    Deno.args;
+if (!release) throw new Error("No release specified!");
+
+// Rewrite imports from .deno.ts to .web.ts
+const cache = createCache();
+const load = (specifier: string) => {
+    if (specifier.endsWith(".deno.ts")) {
+        const baseLength = specifier.length - ".deno.ts".length;
+        specifier = specifier.substring(0, baseLength) + ".web.ts";
+    }
+    return cache.load(specifier);
```

Comment:
```suggestion
    return cache.load(specifier.replace(/\.deno\.ts$/, ".web.ts"));
```

Reply:
Do you think that code is more readable?